org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/workspace/internal/WtpClasspathUpdater.java

```diff
@@ -42,13 +43,15 @@
     private static final String NON_DEPLOYMENT_ATTRIBUTE = "org.eclipse.jst.component.nondependency";
 
     public static void update(IJavaProject javaProject, OmniEclipseProject project, SubMonitor progress) throws JavaModelException {
+      if (Platform.getBundle("org.eclipse.wst.common.core") != null) {
```

Comment:
❌ move the check to the `WorkspaceOperations.isWtpInstalled()`  method in order to make it testable.

Reply:
ok thanks for you suggestions, I will proceed on monday. 
The formatter is not automatically set with oomph? can you please give me a link to the correct formatter?

Reply:
https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.jdt.doc.user%2Freference%2Fpreferences%2Fjava%2Fcodestyle%2Fref-preferences-formatter.htm
https://www.jetbrains.com/help/idea/configuring-code-style.html

Reply:
Yes, the Buildship Oomph setup contains the formatting configuration.

Reply:
Sorry - I don't have it yet... The link of @frontrider leads me to the eclipse formatter preferences mechanism - ok. There are three options I could choose (Eclipse [built-in], Eclipse 2.1 [built-in] and Java-Conventions [built-in]. I think none of this tree options is the right one? 
I have set up my workspace with oomph. Which formatter do I have to choose? Is it somewhere commited to git in the projects?

Reply:
Oomph should set up a new Java formatter called `Buildship`
<img width="920" alt="screen shot 2018-07-24 at 15 27 59" src="https://user-images.githubusercontent.com/419883/43141443-2f2fee2e-8f56-11e8-9ca6-66c8b6e00062.png">
If it doesn't work for you, then just ignore this for this PR. I'll look into the Oomph setup why it's not working for you.

Reply:
Sorry, doesn't work, I don't have the buildship formatter in place. I tried to adapt the format manually best possible, don't now if it is 100% exact... So I changed the isWtpInstalled thing and added a second test...
If you have any requests, please call. 
If you need more info to fix the formatter thing, also call. 

Cheers
Markus